tasks/task02/task02.py

```diff
@@ -0,0 +1,59 @@
+import math
+print("Welcome to the triangle area calculation tool.")
+menu = '''Menu:
+1. Calculate triangle area by base and height
+2. Calculate triangle area by 2 sides and angle between them
+3. Exit
+'''
+while True:
+    print(menu)
+    try:
+        p = int(input("Enter menu item number: "))
+    except ValueError:
+        print("Incorrect input, please input a number")
+    else:
+        if p == 3:
+            break
+        elif p == 1:
+            s = input("Enter base and height: ")
+            parts = s.split(" ")
```

Comment:
splitting by space is a default behavior if no arguments passed, so you can omit `" "` here